cli/command/stack/kubernetes/conversion.go

```diff
@@ -116,25 +116,31 @@ func (t tasksBySlot) Less(i, j int) bool {
 	return t[j].Meta.CreatedAt.Before(t[i].CreatedAt)
 }
 
+const (
+	publishedServiceSuffix      = "-published"
+	publishedOnRandomPortSuffix = "-random-ports"
```

Comment:
nit: There was some discussion around naming of this; as they're not fully "random", but "selected from an ephemeral range"

Reply:
Ok, good to know, but unfortunately these names are defined in the already shipped compose controller...

Reply:
Yeah, probably not a big issue, it's just that they are not really "random" (could be sequential), so "ephemeral" is possibly a better description.